snippets/oadp-ocp-compat.adoc

```diff
@@ -0,0 +1,13 @@
+
+//This snippet appears in the following assemblies:
+//
+// * .../backup_and_restore/backing_up_and_restoring/installing/about-installing-oadp.adoc
+// * .../backup_and_restore/index.adoc
+
+:_content-type: SNIPPET
+[NOTE]
+====
+OADP 1.0._x_ does not support CSI backup on OCP 4.11+. OADP 1.0._x_ includes Velero 1.7._x_ and expects the API group `snapshot.storage.k8s.io/v1beta1`, which is not present on OCP 4.11+.
```

Comment:
Very small nit: "4.11+". Our style guides specify "later" or "later versions" rather than the "+".